yelp_package/dockerfiles/itest/k8s/wait_paasta_api.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+count=1
+
+host="("
+shift
+cmd="$@"
+
+until [ $(curl -o /dev/null -s -w %{http_code} http://127.0.0.1:${PAASTA_API_PORT}/v1/version) = 200 ]; do
+      >&2 echo "PaaSTA API is unavailable."
+        sleep 1
+        count=$((${count}+1))
+        if [ ${count} -gt 9 ]; then
+            echo "Timeout after trying ${count} times"
+            exit 1
+        fi
+    done
+
+    >&2 echo "PaaSTA API is up."
+    exec $cmd
+")"
```

Comment:
what's up with this? won't this result in

```
-bash: ): command not found
```
?

(oh, the `exec` on the previous line prevents this line from ever running. Still, what's this doing here?)

Reply:
Oops. I just realized one of my vim plugins (I don't know which one) converted the script I copied to this werid thing. It worked well so I never notice.